lessons/unit1/dictionaries/README.md

```diff
@@ -1,14 +1,140 @@
-# Standards
-* Master Dictionaries
-* Master Collections
+### Dictionaries
+---
 
-# Objectives
-Students will be able to:
+### Objectives
 * Declare and use Dictionaries 
 * Understand common uses of Dictionaries
 * Contrast and compare Dictionaries with Arrays
 
-# Resources
-Swift Programming: The Big Nerd Ranch Guide, Chapter 10
+### Readings
+1. Swift Programming: The Big Nerd Ranch Guide, Chapter 10
+1. Apple's [Swift Language Reference, Collections](https://developer.apple.com/library/ios/documentation/Swift/Conceptual/Swift_Programming_Language/CollectionTypes.html#//apple_ref/doc/uid/TP40014097-CH8-ID105)
 
-Apple's [Swift Language Reference, Collections](https://developer.apple.com/library/ios/documentation/Swift/Conceptual/Swift_Programming_Language/CollectionTypes.html#//apple_ref/doc/uid/TP40014097-CH8-ID105)
\ No newline at end of file
+#### Vocabulary
+1. **Dictionary** - an unordered collection of key-value associations.
```

Comment:
Two things. I've been debating this but came down on going with lower case on these vocab words. Don't worry about going back anywhere but fix these and please also copy them into the vocabulary file under resources.